### PR TITLE
Allow sending partial coverage reports

### PR DESCRIPTION
**Pull request by DReigada**

### CODE REVIEW

**Comment by DReigada** [comment]
Depends on this PR: codacy/codacy-api-scala#15

File: README.md [hunk 1]
```diff
@@ -35,6 +35,8 @@ Then set it in your terminal, replacing %Project_Token% with your own token:
 export CODACY_PROJECT_TOKEN=%Project_Token%
 ```
 
+You can also use the option `--project-token` or `-l` to set it.
```
[the remark this answers is not in the document]
Comment (DReigada) [reply README.md]: It's defined here: https://github.com/codacy/codacy-coverage-reporter/pull/70/files#diff-08be3df5090c7d0eae4048cad8e49b5bR35

File: README.md [hunk 2]
```diff
@@ -58,15 +60,15 @@ Codacy automatically detects the CommitUUID from several sources:
 
 * You may want to enforce a specific commitUUID with:
 ```
-codacy-coverage-reporter -l Java --commitUUID "mycommituuid" -r coverage.xml
+codacy-coverage-reporter report -l Java --commit-uuid "mycommituuid" -r coverage.xml
```
[the remark this answers is not in the document]
Comment (DReigada) [reply README.md]: Yes, we need to release a major version. The tool should have commands to distinguish between different actions (final notification vs sending report)

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (rtfpessoa) [reply README.md]: That is why I am asking, why is the -l mixed with the project-token?

[the remark this answers is not in the document]
Comment (DReigada) [reply README.md]: You are right, it was a mistake. Going to fix it

**Comment by rtfpessoa** [comment]
Check this https://raw.githubusercontent.com/codacy/codacy-duplication-phpcpd/29fa1b261e224e372597247878144184fcf4be3e/circle.yml to fix the build.

**Pull request merged**